StreetSmartArcMap/Forms/StreetSmartConfigurationForm.cs

```diff
@@ -142,7 +144,15 @@ private void LoadAPISettings()
 
         private void LoadGeneralSettings()
         {
-            nudOverlayDrawDistance.Value = Config.OverlayDrawDistanceInMeters;
+            var selectedSRS = (SpatialReference)cbCycloramaSRS.SelectedItem;
+            switch (selectedSRS.Units)
+            {
+                case "ft":
+                    nudOverlayDrawDistance.Maximum = (decimal)Math.Round((double)nudOverlayDrawDistance.Maximum * 3.280839895, 0);
```

Comment:
What if the selected srs is meters. Then I think the value is no longer entered correctly. Since this case only has a feet version. And this line has been deleted:
nudOverlayDrawDistance.Value = Config.OverlayDrawDistanceInMeters;

Reply:
nudOverlayDrawDistance.Value = Config.OverlayDrawDistanceInMeters; will override the calculated draw distance based on ft or m. If ft it will be overridden by 30 always.

Reply:
But what if you change your overlay draw distance to 60 meters, you save your settings. you restart ArcMap. Is there 60 meters or 30 meters in this field? the correct answer should be 60 meters.